release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1030,6 +1030,16 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-cloud-compute-bug-fixes_{context}"]
 ==== Cloud Compute
 
+* Previously, a machine controller failed to save the {vmw-full} task ID of an instance template clone operation. This caused the machine to go into the `Provisioning` state and to power off. With this release, the {vmw-full} machine controller can detect and recover from this state. (link:https://issues.redhat.com/browse/OCPBUGS-1735[*OCPBUGS-1735*])
+
+* Previously, the `machine-api` Operator reacted when it deleted a server that was in an `ERROR` state. This happened because the server did not pass a port list. With this release, deleting a machine stuck in an `ERROR` state does not cause this issue anymore. (link:https://issues.redhat.com/browse/OCPBUGS-34155[*OCPBUGS-34155*])
```

Comment:
```suggestion
* Previously, the `machine-api` Operator reacted when it deleted a server that was in an `ERROR` state. This happened because the server did not pass a port list. With this release, deleting a machine stuck in an `ERROR` state does not cause an Operator reaction. (link:https://issues.redhat.com/browse/OCPBUGS-34155[*OCPBUGS-34155*])
```

Reply:
:heavy_check_mark: